Modules/Filtering/ImageStatistics/include/itkLabelOverlapMeasuresImageFilter.hxx

```diff
@@ -375,15 +375,19 @@ LabelOverlapMeasuresImageFilter<TLabelImage>::GetFalsePositiveError() const -> R
 {
   RealType numerator = 0.0;
   RealType denominator = 0.0;
+
+  auto nVox = this->GetInput(0)->GetBufferedRegion().GetNumberOfPixels(); // TP+FP+FN+TN
```

Comment:
This should be `GetLargestPossibleRegion`. This make is always correct even if the input's buffer is used by another filter or if this filter is converted to modern streamable `ImageSink` base.

Reply:
Thanks for suggestion, force-push makes the change.